cmd/kubectl-directpv/list.go

```diff
@@ -66,10 +69,25 @@ func validateGetCmd() error {
 	if err := validateNodeArgs(); err != nil {
 		return err
 	}
-
 	if err := validateDriveNameArgs(); err != nil {
 		return err
 	}
+	if err := validateLabelArgs(); err != nil {
+		return err
+	}
 
 	return nil
 }
+
+func humanizeLabels(labels map[string]string) string {
+	var labelsArray []string
+	for k, v := range labels {
+		if !strings.HasPrefix(k, consts.GroupName) {
```

Comment:
1. Rename this function as `labelsToString()`
2. Directly call `strings.TrimPrefix()`